tests/tests/test_dbt.py

```diff
@@ -7,22 +7,32 @@
 from dbt.cli.main import dbtRunner, dbtRunnerResult
 
 
+@pytest.mark.parametrize("model_name,run_success,query_result",
+                         [
+                             ("table_materialization", True, 10),
+                             pytest.param("view_materialization", True, 10, marks=pytest.mark.xfail),
```

Comment:
to summarize: this now works for tables but not views?

Reply:
Yes, exactly.